npm/install.js

```diff
@@ -17,7 +17,9 @@ try {
 
 var platformPath = getPlatformPath()
 
-if (installedVersion === version && fs.existsSync(path.join(__dirname, platformPath))) {
+var electronPath = process.env.ELECTRON_OVERRIDE_DIST_PATH || path.join(__dirname, 'dist', platformPath)
```

Comment:
I'm not sure how this fixes the linked issue, the linked issue is actually caused by `electron-download` not this module afaik

Reply:
@MarshallOfSound 
The error is not coming from `electron-download` it's from `extract-zip` code below
![screen shot 2018-08-18 at 9 41 20 am](https://user-images.githubusercontent.com/596478/44295666-5d2b9c00-a2cb-11e8-8a00-b46a78d025df.png)

This is because of `fs.existsSync(path.join(__dirname, platformPath))` is always false as the electron path from the `npm install` is `dist/Electron.app/Contents/MacOS/Electron`.

If the path check is false `process.exit(0)` won't execute and  `extract-zip` is trying to symlink the path which already exists.

Reply:
Ah I see, the actual fix here is the inclusion of `dist`, the process env var is just for consistency with the download script